README.md

```diff
@@ -5,3 +5,8 @@
 - This one is correct
 - This needs a fix
 - This one is correct too
+
+#### markdown-fixes:
+something something
+ -  
+  - fasdf
```

Comment:
Codacy has a fix for the issue: [Lists should be surrounded by blank lines](https://app.codacy.com/gh/troubleshoot-codacy/markdown-fixes/pullRequest?prid=9551052)
```suggestion

  - fasdf
```